libs/ml/include/ml/ops/estimation/charge_constants.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
```

Comment:
maybe this file should be moved to ml/charge_estimation/ops/constants.hpp?
Otherwise we're going to end up with many "estimation" subfolders all over the place.

Reply:
well, if we have a plan to estimate not only Ops but other ML stuff, it is reasonable, I'll move it

Reply:
done, I've splitted it for clearer dependency tree